index.html

```diff
@@ -1,287 +1,299 @@
-                  <h2 class="project-title">Profesional Art Printing Data</h2>
-                  <h2 class="desktop-title project-title-desktop-1">Profesional Art <br> Printing Data More</h2>
-                  <p class="project-discription">A daily selection of privately personalized reads; no accounts 
-                    or sign-ups required. has been the industry's standard</p>
-                    <ul class="technologies">
-                     <li>html</li>
-                     <li>bootstrap</li>
-                     <li>Ruby</li>
-                   </ul>
 
-               <button class="button">See Project</button>
-
-                </div>
-                <div class="project-card card-2">
-                  <h2 class="project-title">Profesional Art Printing Data</h2>
-                  <h2 class="desktop-title project-title-desktop-1">Data Dashboard <br> Healthcare</h2>
-                  <p class="project-discription">A daily selection of privately personalized reads; no accounts 
-                    or sign-ups required. has been the industry's standard</p>
-                    <ul class="technologies">
-                     <li>html</li>
-                     <li>bootstrap</li>
-                     <li>Ruby</li>
-                   </ul>
-
-               <button class="button">See Project</button>
-
-                </div>
-                <div class="project-card card-3">
-                  <h2 class="project-title">Profesional Art Printing Data</h2>
-                    <h2 class="desktop-title project-title-desktop-2">Website Protfolio </h2>
-                    <p class="project-discription">A daily selection of privately personalized reads; no accounts 
-                      or sign-ups required. has been the industry's standard</p>
-                      <ul class="technologies">
-                       <li>html</li>
-                       <li>bootstrap</li>
-                       <li>Ruby</li>
-                     </ul>
-  
-                 <button class="button">See Project</button>
-
-                </div>
+<body>
+  <header class="header">
+    <a href="index.html" id="logo">
+      <h1 class="name">Biruk Bereka</h1>
+    </a>
+    <nav class="nav-bar">
+      <div class="menu-icon" id="menu"><img src="./Icons/Icon-Menu.png" alt=""></div>
+      <div class="nav-items" id="menu-list">
+        <ul id="list-items">
+          <li id="cancel" class="cancelIcon"><img src="Icons/Icon-Cancel.png" alt=""></li>
+          <li class="menu-item"><a href="#portfolio">Portfolio</a></li>
+          <li class="separtor"></li>
+          <li class="menu-item"><a href="#about">About</a></li>
+          <li class="separtor"></li>
+          <li class="menu-item"><a href="#contact-section">Contact</a></li>
+          <li class="separtor"></li>
+          <li id="mail-icon"><a class="mail" href="#mail"><img src="./Icons/Union.png" alt=""></a></li>
+        </ul>
+      </div>
+    </nav>
+  </header>
```

Comment:
- [ ] The functionality of the mobile menu works fine. But you might need to fix up small design alignments. Your mobile menu has white space at the top and the nav links in the menu are slightly off as per your chosen design. Please check if you have some margins or padding that are causing this issue.

Your Design | Template Design |
:---------------:|:-----------------------:|
![a2](https://user-images.githubusercontent.com/39523312/157291883-24651669-5357-472e-a43e-3ddb2f81593c.png)| ![a1](https://user-images.githubusercontent.com/39523312/157291893-efdbabf6-cf8f-416c-acf6-e44d6e3ee2c2.png)